recipe/meta.yaml

```diff
@@ -8,23 +8,25 @@ package:
 source:
   url: https://github.com/conda/{{ name }}/archive/{{ version }}.tar.gz
   sha256: b0014986321e77a36d8fe24827698aae076500c817133a700608bd677b77a998
+  patches:
+    # Patch for unittest.makeSuite (removed in Python 3.13+)
+    - patches/0001-GH-100-Replace-unittest.makeSuite-with-unittest.defaultTest.patch
 
 build:
-  number: 1 # trigger
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 2
+  script: {{ PYTHON }} -m pip install . --no-deps -vv --no-build-isolation
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
```

Comment:
You can get rid of `# [build_platform != target_platform]`

Reply:
I can but I don't think it affects our builds if it stays. I intentionally left it because it's been in the feedstock for over 4 years and is compatible with conda-forge. However, if there are still objections, I'm okay with removing it.

Reply:
I would leave it. Just to minimize the changes to the recipe, and in case we starts cross-compiling again later on.

Reply:
Okay, thanks!